pkg/ebpf/ingress_node_firewall_events.go

```diff
@@ -30,6 +30,18 @@ func (infc *IngNodeFwController) ingressNodeFwEvents() {
 		log.Printf("Failed creating perf event reader: %q", err)
 		return
 	}
+	logFile, ok := os.LookupEnv("EVENT_LOGGING_FILE")
```

Comment:
Do we need to worry about consuming too much disk space logging to file which would eventually kill our program? Is there software that would allow us to set a limit?

Reply:
something like this https://kubernetes.io/docs/concepts/cluster-administration/logging/#cluster-level-logging-architectures ?
I will need to look at this in separate PR , ok ?